packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -236,6 +237,23 @@ export const FLAGS: Record<string, FlagInfo> = {
     compatibilityWithOldBehaviorMd: 'You can pass `desiredCount: 1` explicitly, but you should never need this.',
   },
 
+  //////////////////////////////////////////////////////////////////////
+  [ECS_ADD_SECURITY_GROUP]: {
```

Comment:
```suggestion
  [ECS_ADD_SECURITY_GROUPS_TO_ASG_CAPACITY_PROVIDERS]: {
```
let's call this feature flag something more descriptive

Reply:
Got it and changed .... you can see in the latest commit